phoenix-core/src/it/java/org/apache/phoenix/end2end/AlterTableIT.java

```diff
@@ -852,6 +854,39 @@ public void testDropMultipleColumns() throws Exception {
         conn1.close();
     }
 
+    @Test
+    public void testAlterTableOnGlobalIndex() throws Exception {
+        try (Connection conn = DriverManager.getConnection(getUrl())) {
+            conn.setAutoCommit(false);
+            Admin admin = conn.unwrap(PhoenixConnection.class).getQueryServices().getAdmin();
+            Statement stmt = conn.createStatement();
```

Comment:
Can we reuse `stmt` throughout this test rather than calling `conn.createStatement()` again? Also, you can move this inside the try-with-resources block so the Statement object also gets closed at the end